scripts/agent/README.md

```
@@ -0,0 +1,402 @@
+
+# Compile for Local RAG (Ollama required)
+bun run compile:local
+# → generates LanceDB vector index
+
+# Full compile (AI + Local RAG)
+bun run compile
+```
+
+### 4. Run Hybrid Mode Testing
+
+**Step 1: Run Local Agent (Challenger)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)